index.bs

```diff
@@ -2984,6 +2985,13 @@ browsingContext.CssLocator = {
    value: text
 }
 
+browsingContext.ContextLocator = {
```

Comment:
I still find it puzzling when naming it `ContextLocator` but then returning a `Node` which is the container element of a context. Why was it changed from `ContainerLocator`?

Reply:
Changed on request by @jgraham because the locator name reflects the criteria for search and not the return value. I.e., browsingContext.ContextLocator locates by a context selector,  browsingContext.CssLocator locates by a CSS selector.

Reply:
I’m still not entirely sold on this API, but if @jgraham is okay with it, we can move forward.